tests/ha/ha_cluster_init.pm

```diff
@@ -25,7 +25,6 @@ sub run {
     my $sbd_device    = get_lun;
     my $unicast_opt   = get_var("HA_UNICAST") ? '-u' : '';
     my $quorum_policy = 'stop';
-    my $join_timeout  = 60;
```

Comment:
As `$join_timeout` is already declared in `hacluster.pm` I think it's not so useful here, right? And this variable seems not used in `ha_cluster_init.pm`.

Reply:
I agree because `$join_timeout` is now a global variable.

Reply:
Yes, that's why I'm removing it from that module.